.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] | "
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screnshots / Videos**
+If applicable, add screenshots or videos to help explain your problem.
+
+**Where do the issue happen? (please complete the following information)**
```

Comment:
just a grammatical error, it should be where "does" the issue happen?

Reply:
Thanks! Fixed it